### PR TITLE
Improved static UI of select product and select facility page(#28zvc40)

### PR DESCRIPTION
**Pull request by azkyakhan**
select-product-page

select-facility-page

### CODE REVIEW

File: src/views/SelectProduct.vue [hunk 1]
```diff
@@ -157,14 +157,25 @@ export default defineComponent({
...
-}
 .section-grid {
   grid-template-columns: repeat(auto-fill, 200px);
 }
 
 @media (min-width: 991px) {
+  .find {
+    grid: "search  filters"
+          "main    main"
+          / 440px 400px;
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:168]: I don't think we need to set any explicit values on columns right now. By default I think it should split the space in half. We can manage width of search and select on the elements themselves.

File: src/views/SelectProduct.vue [hunk 2]
```diff
@@ -157,14 +157,25 @@ export default defineComponent({
...
+          "main    main"
+          / 440px 400px;        
+    justify-content: space-between;   
+    row-gap: var(--spacer-xl);
+    margin: var(--spacer-lg) var(--spacer-base);               
+  }
+
+  main {
+    padding: var(--spacer-sm) 0;
+    border-top: 1px solid var(--ion-color-medium)
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:176]: we can introduce global border variable here

File: src/views/SelectProduct.vue [hunk 3]
```diff
@@ -157,14 +157,25 @@ export default defineComponent({
...
+  }
+  
   .action {
     position: absolute;
     bottom: 25%;
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:181]: change this to a fixed value of px and make this a global class

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/SelectProduct.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: I have removed explicit values on columns.

File: src/views/SelectProduct.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: Ok Sir, I have replaced this value with border varibale.

File: src/views/SelectProduct.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: I have changed the bottom value in px and made this global class and removed styling from the select product and select facility page.

**Comment by azkyakhan** [comment]
select-product-page

select-facility-page

**Comment by ymaheshwari1** [comment]
Closing as related issue is closed

**Pull request closed without merging**